src/a09-coverage/test/solution.test.js

```diff
@@ -0,0 +1,32 @@
+import { average, sum, product } from '../src/index.js'
+import { test } from 'node:test'
+import assert from 'node:assert'
+
+test('sum', () => {
+  assert.deepStrictEqual(sum([1, 2, 3]), 6, 'sum of [1, 2, 3] is 6')
+  assert.deepStrictEqual(sum([]), 0, 'sum of empty array is 0')
+  assert.throws(
+    () => sum('abc'),
+    { message: 'Input must be an array of numbers' },
+    'throws error for non-array input in sum'
```

Comment:
can we please get rid of error messages in assertions? they're fairly uncommon

Reply:
Sure :) Will remove it from here and in case we want to remove them from all the chapters, I'll issue another cleanup PR

Reply:
if we really want to show them, we can show them once and then not anymore. I'm also fine if we don't want to show them at all. although they are a feature of the test runner (or more specifically assertions) they're so little used that I don't think we're omitting essential information.

Reply:
Agreed! Will create a cleanup PR to keep the messages just in one chapter